### PR TITLE
Dev/issue directive

### PR DESCRIPTION
**Pull request by abierbaum**

### CODE REVIEW

File: src/app/issue/issue_ctrl.js [hunk 1]
```diff
@@ -1,11 +1,14 @@
-angular.module('GitKan.board')
+mod = angular.module('GitKan.issue', []);
```
Comment (my8bird) [inline src/app/issue/issue_ctrl.js]: BUG: `var = mod`

File: src/app/board/columns/issue_columns.tpl.html [hunk 2]
```diff
@@ -26,7 +26,7 @@ <h1 class="column-header">{{colCtrl.columnName}}</h1>
                               ng-model="colCtrl.issues" >
         <div class="card_wrapper"
             ng-repeat="issue in colCtrl.issues | filter:main.cardSearchText">
-          <div ng-include="'board/columns/issue.tpl.html'"></div>
+          <div tr-issue-card issue="issue"></div>
```
Comment (my8bird) [inline src/app/board/columns/issue_columns.tpl.html:29]: ???

``` html
<tr-issue-card issue="issue">
```

File: src/app/issue/issue_ctrl.js [hunk 3]
```diff
@@ -31,5 +34,18 @@ angular.module('GitKan.board')
...
+
+   this.init($scope.$parent.issue);
+});
+
+mod.directive('trIssueCard', function() {
+   return {
+      restrict: 'EA',
+      replace: true,
+      templateUrl: "issue/issue.tpl.html",
+      scope: {
```
Comment (my8bird) [inline src/app/issue/issue_ctrl.js:46]: controllerAs: 'IssueControlelr as issueCtrl'

ng-init: 'issueCtrl.init(issue)'

File: src/app/issue/issue_ctrl.js (hunk 1 above)
Comment (abierbaum) [inline src/app/issue/issue_ctrl.js]: done.

**Pull request merged**